lib/Smokeping/probes/base.pm

```diff
@@ -340,7 +340,71 @@ DOC
 }
 
 sub targetvars {
-	return {_mandatory => []};
+	return {_mandatory => [],
+		influx_generic1 => {
+			_re => '.*',
+			_example => '',
+			_doc => <<DOC,
+This is a tag (generic1) that will be sent to influxdb and has no impact on the probe measurement
+DOC
+		},
+		influx_generic2 => {
+			_re => '.*',
+			_example => '',
+			_doc => <<DOC,
+This is a tag (generic2) that will be sent to influxdb and has no impact on the probe measurement
+DOC
+		},
+		influx_generic3 => {
+			_re => '.*',
+			_example => '',
+			_doc => <<DOC,
+This is a tag (generic3) that will be sent to influxdb and has no impact on the probe measurement
+DOC
+		},
+		influx_generic4 => {
+			_re => '.*',
+			_example => '',
+			_doc => <<DOC,
+This is a tag (generic4) that will be sent to influxdb and has no impact on the probe measurement
+DOC
+		},
+		influx_generic5 => {
+			_re => '.*',
+			_example => '',
+			_doc => <<DOC,
+This is a tag (generic5) that will be sent to influxdb and has no impact on the probe measurement
+DOC
+		},
+		influx_generic6 => {
+			_re => '.*',
+			_example => '',
+			_doc => <<DOC,
+This is a tag (generic6) that will be sent to influxdb and has no impact on the probe measurement
+DOC
+		},
+		influx_generic7 => {
+			_re => '.*',
+			_example => '',
+			_doc => <<DOC,
+This is a tag (generic7) that will be sent to influxdb and has no impact on the probe measurement
+DOC
+		},
+		influx_generic8 => {
+			_re => '.*',
+			_example => '',
+			_doc => <<DOC,
+This is a tag (generic8) that will be sent to influxdb and has no impact on the probe measurement
+DOC
+		},
+		influx_generic9 => {
+			_re => '.*',
+			_example => '',
+			_doc => <<DOC,
+This is a tag (generic9) that will be sent to influxdb and has no impact on the probe measurement
+DOC
+		},
```

Comment:
this can be written in a much simpler and more generic way ... 

where are you doing the actually sending of this ? 

the naming seems very 'generic' how about `influx_meta_\d`  or something like that ?

an actual example would also be very nice

```suggestion
	return {_mandatory => [],
		'/influx_generic\d+/' => {
			_re => '.+',
			_example => '',
			_doc => <<DOC,
The content of this variable will be sent to influxdb and has no impact on the probe measurement
DOC
		},
```

Reply:
Didn't know you could use regex there. Yes, you're right, it sounds better as you suggested. Hope to refactor it soon. The data gets sent to the influxdb as tags and can help with filtering/enriching the measurements sent.

Reply:
The data gets parsed here: https://github.com/mad-ady/SmokePing/blob/af6f9ee17afb3f8995560bcdf6e0cc4c8d25469e/lib/Smokeping.pm#L2191
... and prepared for insertion here: https://github.com/mad-ady/SmokePing/blob/af6f9ee17afb3f8995560bcdf6e0cc4c8d25469e/lib/Smokeping.pm#L2218

Reply:
in the code you mention the danger of name clashes ... how about adding a prefix to all tags inserted manually ?

Reply:
You're right. Your suggestion allows me to add any influxdb tag names (which is what I wanted all along). I also prefix all the custom tags with the string tag_ when inserting them in influxdb, thus fixing the name clash, but also breaking the current name scheme... I will need to make some adjustments on the Grafana side (and drop old tags). Hopefully it won't impact many people...

Reply:
What about configuration examples? I have a blog post about it (which I will update to reflect the current changes): http://adrianpopagh.blogspot.com/2020/06/smokeping-influxdb-export-docker-slaves.html, but not sure if there is more I can add about it to your project (convert some of the information in a readme-influxdb?)